python_modules/dagster/dagster/check/__init__.py

```diff
@@ -309,6 +309,22 @@ def opt_list_param(obj_list, param_name, of_type=None):
     return _check_list_items(obj_list, of_type)
 
 
+def opt_nullable_list_param(obj_list, param_name, of_type=None):
```

Comment:
add some tests to these to demonstrate different behavior,  get code coverage, etc